src/googleAnalytics.js

```diff
@@ -0,0 +1,14 @@
+import ReactGA from 'react-ga';
+
+export default {
+  hasGABeenInit: false,
+  initGA: function() {
```

Comment:
I think we can use a shorter syntax for method definitions on objects initializers. Given the following code:

```js
var obj = {
  foo: function() {
    /* code */
  },
  bar: function() {
    /* code */
  }
};
```

You are now able to shorten this to:
```js
var obj = {
  foo() {
    /* code */
  },
  bar() {
    /* code */
  }
};
```

MDN Doc Link: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions

Reply:
@arecvlohe You're right. I'll update the PR for this.